Makefile.am

```diff
@@ -163,7 +167,9 @@ EXTRA_DIST = COPYING COPYING.libcrun README.md NEWS SECURITY.md rpm/crun.spec au
 	krun.1.md krun.1 \
 	lua/luacrun.rockspec
 
+if BUILD_TESTS
 UNIT_TESTS = tests/tests_libcrun_utils tests/tests_libcrun_ring_buffer tests/tests_libcrun_errors tests/tests_libcrun_intelrdt
+endif
```

Comment:
does it affect `make dist` and what files are included in the tarball if we build without glibc-static?

Reply:
It doesn´t affect make dist (tested on fedora42). autotools is smart enough to comment out only the build section for the tools and keep the _SOURCE bits for *_DIST sections.

Just one example:

```
$ grep tests_libcrun_utils Makefile
am__tests_tests_libcrun_utils_SOURCES_DIST =  \
        tests/tests_libcrun_utils.c
```

and just to be 100% safe. Done 2 make dist, one with glib-static and one without:

```
crun-without/tests$ ls -1 | wc -l
34

crun-with/tests$ ls -1 | wc -l
34

fabbione@mazikeen:~/work/containers$ diff -Naurd crun-without/ crun-with
fabbione@mazikeen:~/work/containers$
```